coffee-chats/src/main/java/com/google/step/coffee/entity/TimeSlot.java

```diff
@@ -0,0 +1,21 @@
+package com.google.step.coffee.entity;
+
+import java.time.ZonedDateTime;
+
+public class TimeSlot {
+  private final ZonedDateTime datetimeStart;
+  private final int duration;
```

Comment:
It's not clear from the code which units the duration uses. Maybe `java.time.Duration`?